src/Impl/GuzzleHttpRequester.php

```diff
@@ -41,18 +40,15 @@ public function executePost(
     }
 
     /**
+     * @return MessageStatus[]
      * @throws StreamxClientException
      */
-    private function handleResponse(ResponseInterface $response): SuccessResult
+    private function handleResponse(ResponseInterface $response): array
     {
         $statusCode = $response->getStatusCode();
 
         if ($statusCode == 202) {
-            $messageStatus = $this->parseMessageStatus($response);
-            if ($messageStatus->getSuccess() != null) {
-                return $messageStatus->getSuccess();
-            }
-            throw $this->streamxClientExceptionFrom($messageStatus->getFailure());
```

Comment:
This if/return/throw operations are now moved to the send(single_message) method